### PR TITLE
Athenian temple and wonder

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/structures/athen/temple.xml [hunk 1]
```diff
@@ -7,6 +7,10 @@
   <Identity>
     <Civ>athen</Civ>
     <SpecificName>Naos</SpecificName>
+    <History>
+        Athens was extremely religious. The city state itself was named in honor Athena after she, as the Greek myth says, created the olive tree for them. She was their primary deity. That said, Athens worshiped all gods the Greek pantheon and many besides.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/structures/athen/temple.xml:11]:
```suggestion
        Athens was extremely religious. The city state itself was named in honor  of Athena after she, as the Greek myth says, created the olive tree for them. She was their primary deity. That said, Athens worshiped all gods of the Greek pantheon and many besides.
```

File: 0-ad-history-mod-package/simulation/templates/structures/athen/wonder.xml [hunk 2]
```diff
@@ -7,6 +7,10 @@
   <Identity>
     <Civ>athen</Civ>
     <SpecificName>Parthenōn</SpecificName>
+    <History>
+        The Parthenon was the most magnificent temple in all of Ancient Greece. It was built after the Greco-Persian, replacing an older structure that was destroyed by the Persians. It was decorated with many sculptures, the biggest being the Athena Parthenose. Constructed by Pheidias, the Athena Parthenose was around 38 feet (11.5 meters) tall, and plated with gold and ivory.
+        Unfortunately, the Parthenon has not aged well. Over the years the structure has been converted both into a Greek Orthodox church, a Mosque, and an ammunition dump, that exploded, destroyed the buildings center. It still stands on the Athenian acropolis, a ruin with most of its statues stolen or destroyed.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/structures/athen/wonder.xml:12]:
```suggestion
        Unfortunately, the Parthenon has not aged well. Over the years the structure has been converted both into a Greek Orthodox church, a Mosque, and an ammunition dump, that exploded, destroyed the building's center. It still stands on the Athenian acropolis, a ruin with most of its statues stolen or destroyed.
```

File: 0-ad-history-mod-package/simulation/templates/structures/athen/temple.xml [hunk 3]
```diff
@@ -7,6 +7,10 @@
   <Identity>
     <Civ>athen</Civ>
     <SpecificName>Naos</SpecificName>
+    <History>
+        Athens was extremely religious. The city state itself was named in honor  of Athena after she, as the Greek myth says, created the olive tree for them. She was their primary deity. That said, Athens worshiped all gods of the Greek pantheon and many besides.
+        Perhaps, one of the most amusing stories of Athenian was at the battle of Marathon where Athens, facing superior odds, vowed to sacrifice a she-goat to Artemis for every Persian they slew. They killed 6,400 Persians. They did not have that many she-goats on hand, so decided that they would sacrifice 500 goats per year, and did so for nearly a century.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/structures/athen/temple.xml:12]:
```suggestion
        Perhaps one of the most amusing stories of Athenian was at the Battle of Marathon, where Athens, facing superior odds, vowed to sacrifice a she-goat to Artemis for every Persian they slew. They killed 6,400 Persians. They did not have that many she-goats on hand, so decided that they would sacrifice 500 goats per year, and did so for nearly a century.
```

File: 0-ad-history-mod-package/simulation/templates/structures/athen/wonder.xml [hunk 4]
```diff
@@ -7,6 +7,10 @@
   <Identity>
     <Civ>athen</Civ>
     <SpecificName>Parthenōn</SpecificName>
+    <History>
+        The Parthenon was the most magnificent temple in all of Ancient Greece. It was built after the Greco-Persian, replacing an older structure that was destroyed by the Persians. It was decorated with many sculptures, the biggest being the Athena Parthenose. Constructed by Pheidias, the Athena Parthenose was around 38 feet (11.5 meters) tall, and plated with gold and ivory.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/structures/athen/wonder.xml:11]:
```suggestion
        The Parthenon was the most magnificent temple in all of Ancient Greece. It was built after the Greco-Persian Wars, replacing an older structure that was destroyed by the Persians. It was decorated with many sculptures, the biggest being the Athena Parthenose. Constructed by Pheidias, the Athena Parthenose was around 38 feet (11.5 meters) tall and plated with gold and ivory.
```

File: 0-ad-history-mod-package/simulation/templates/structures/athen/wonder.xml [hunk 5]
```diff
@@ -7,6 +7,10 @@
...
+        Unfortunately, the Parthenon has not aged well. Over the years the structure has been converted both into a Greek Orthodox church, a Mosque, and an ammunition dump, that exploded, destroyed the building's center. It still stands on the Athenian acropolis, a ruin with most of its statues stolen or destroyed.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/structures/athen/wonder.xml:12]:
```suggestion
        Unfortunately, the Parthenon has not aged well. Over the years, the structure has been converted both into a Greek Orthodox church, a mosque, and an ammunition dump that exploded and destroyed the building's center. It still stands on the Athenian acropolis, a ruin with most of its statues stolen or destroyed.
```

**Review by indoptogopt — CHANGES_REQUESTED** [review]

**Comment by TheShadowOfHassen** [comment]
Is this OK to merge?

File: 0-ad-history-mod-package/simulation/templates/structures/athen/wonder.xml [hunk 6]
```diff
@@ -7,6 +7,10 @@
   <Identity>
     <Civ>athen</Civ>
     <SpecificName>Parthenōn</SpecificName>
+    <History>
+        The Parthenon was the most magnificent temple in all of Ancient Greece. It was built after the Greco-Persian Wars, replacing an older structure that was destroyed by the Persians. It was decorated with many sculptures, the biggest being the Athena Parthenose. Constructed by Pheidias, the Athena Parthenose was around 38 feet (11.5 meters) tall and plated with gold and ivory.
+        Unfortunately, the Parthenon has not aged well. Over the years, the structure has been converted both into a Greek Orthodox church, a mosque, and an ammunition dump that exploded and destroyed the building's center. It still stands on the Athenian acropolis, a ruin with most of its statues stolen or destroyed.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/structures/athen/wonder.xml:12]:
```suggestion
        Unfortunately, the Parthenon has not aged well. Over the years, the structure has been converted both into a Greek Orthodox church, a mosque, and an ammunition dump that exploded and destroyed the building's center. It still stands on the Athenian acropolis, but it is now a ruin, with most of its statues stolen or destroyed.
```

**Pull request merged**